src/subscription/__tests__/subscribe-test.js

```diff
@@ -77,10 +77,7 @@ describe('Subscribe', () => {
     subscription: SubscriptionType
   });
 
-  it('produces a payload per subscription event', async () => {
-
-    const pubsub = new EventEmitter();
-
+  const createSubscription = pubsub => {
```

Comment:
minor style detail: functions defined at the top level we use `function createSubscription(pubsub) {`